File: Java/1-if-else/src/if_else/Exercicio7.java
package if_else;

import java.util.Scanner;

/*
7- Faça um programa que receba dois números e mostre o maior. Se por acaso, os dois números
forem iguais, imrpima números iguais.
*/
public class Exercicio7 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int numero1, numero2;

        System.out.println("Digite o primeiro número: ");
        numero1 = scan.nextInt();
        System.out.println("Digite o primeiro número: ");
        numero2 = scan.nextInt();

        if (numero1 == numero2){
            System.out.println("Números são iguais");
        } else if (numero1 > numero2){
            System.out.println("Número 1 é o maior");
        } else {
            System.out.println("Número 2 é o maior");
        }
    }
}

File: Python/1-if-else/exercicio7.py
"""
7- Faça um programa que receba dois números e mostre o maior. Se por acaso, os dois números
forem iguais, imrpima números iguais.
"""

print('Digite o primeiro número: ')
numero_1 = int(input())

print('Digite o primeiro número: ')
numero_2 = int(input())

if numero_1 == numero_2:
    print('Números são iguais!')
elif numero_1 > numero_2:
    print('Numero 1 é maior.')
else:
    print('Número 2 é maior')
